app/models/orchestration_template.rb

```diff
@@ -22,6 +22,8 @@ class OrchestrationTemplate < ApplicationRecord
   attr_accessor :remote_proxy
   alias remote_proxy? remote_proxy
 
+  attr_accessor :format
```

Comment:
We are not ready to add this attribute to base class yet, for every subclass needs to implement it.

I suggest to add a reader method to `OrchestrationTemplateCnf` which is stateless because the content can vary. It should be quick enough to detect whether it might be JSON or not. If not we would assume YAML
```
   def format
    content.strip.first == '{' ? 'json' : 'yaml'  # should it be upper case?
  end
```